.github/workflows/ghci.yml

```diff
@@ -91,8 +103,19 @@ jobs:
               ;;
           esac
 
+      - name: Customize compiler
+        if: matrix.api == 'app' && matrix.cc == 'gcc'
+        run: >
+          gcc -dumpspecs
+          | sed 's/-lmsvcrt/-lucrtapp/g; s/-lkernel32/-lwindowsapp/g;
+          s/-ladvapi32//g; s/-lshell32//g; s/-luser32//g'
+          > /tmp/windowsapp.specs
+
       - name: configure
-        run: ./autogen.sh && CC="${{ matrix.cc }}" ./configure
+        env:
+          CC: "${{ matrix.cc }}\
+            ${{ matrix.extra_cflags && ' ' }}${{ matrix.extra_cflags }}"
```

Comment:
I think this can be simplified to the imho more readable:
```
  CC: "${{ matrix.cc }} ${{ matrix.extra_cflags }}"
```
The additional whitespace when `matrix.extra_cflags` is undefined shouldn't be harmful.

Reply:
It’s not harmful, but it’s mildly annoying :-) as it is reflected in `configure` output. But if you think simplicity outweighs that (please confirm), I’ll remove this.

Reply:
nah, if you want a cleaner config-logs it's fine.